packages/vscode-extension/package.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "netsuite-suitecloud-vscode-extension",
```

Comment:
I think we're removing "netsuite" from other module names, probably we should do it here as well